File: app/src/main/java/cc/smartcash/wallet/Services/UserService.java
package cc.smartcash.wallet.Services;


import cc.smartcash.wallet.Models.ApiResponse;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Header;

public interface UserService {

    @GET("user/my")
    Call<ApiResponse> getUser(@Header("Authorization") String auth);
}

File: app/src/main/java/cc/smartcash/wallet/Services/UserService.kt
package cc.smartcash.wallet.Services


import cc.smartcash.wallet.Models.User
import cc.smartcash.wallet.Models.WebWalletRootResponse
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Header

interface UserService {

    @GET("user/my")
    fun getUser(@Header("Authorization") auth: String): Call<WebWalletRootResponse<User>>
}
